magma/backend/verilog.py

```diff
@@ -165,72 +168,69 @@ def compiledefinition(cls):
                 raise Exception(f'Argument {cls.__name__}.{name} of type {type(port)} is not supported, the verilog backend only supports simple 1-d array of bits of the form Array(N, Bit)')
 
 
-    if cls.verilogFile:
-       s = cls.verilogFile
```

Comment:
This is an awkward diff, but it basically removes the `if cls.verilogFile` condition, moves it up and uses a return, and removes the else case (causing the re-indentation and big diff)